hail/python/hail/methods/statgen.py

```diff
@@ -1877,8 +1877,9 @@ def pc_relate(call_expr, min_individual_maf, *, k=None, scores_expr=None,
 
 @typecheck(ds=oneof(Table, MatrixTable),
            keep_star=bool,
-           left_aligned=bool)
-def split_multi(ds, keep_star=False, left_aligned=False):
+           left_aligned=bool,
+           permit_shuffle=bool)
+def split_multi(ds, keep_star=False, left_aligned=False, *, permit_shuffle=False):
```

Comment:
What's the * doing in this case? does it just enforce that `permit_shuffle` has to be defined as a kwarg?

Reply:
yep. I think we should do this more going forward, since it makes back-compatibility easier (argument order doesn't mater)